modules/fast_float/8.0.2/MODULE.bazel

```diff
@@ -0,0 +1,9 @@
+"""fast_float number parsing library: 4x faster than strtod"""
+
+module(
+    name = "fast_float",
+    version = "8.0.2",
+    compatibility_level = 8,
```

Comment:
Does the new major version contain any breaking changes to the API? If not, then raising this level will be more annoying than helpful.